app/views/collections/show.html.erb

```diff
@@ -106,12 +106,12 @@
               <%= link_to('Domains<br />'.html_safe + number_to_human_size(File.size(domains_path(@user.id, @collection.id, @collection.account))), user_collection_download_domains_path(@user.id, @collection.id, @collection.account), class: 'btn btn-primary', 'data-toggle': 'tooltip', 'data-placement': 'top', title: 'A csv file that contains two columns; the domain, and number of occurrences (the distribution of domains within the web archive).') %>
             <% end %>
             <% unless !File.exists? fulltext_path(@user.id, @collection.id, @collection.account) %>
-              <%= link_to('Full Text<br />'.html_safe + number_to_human_size(File.size(fulltext_path(@user.id, @collection.id, @collection.account))), user_collection_download_fulltext_path(@user.id, @collection.id, @collection.account), class: 'btn btn-primary', 'data-toggle': 'tooltip', 'data-placement': 'top', title: 'A csv file that contains four columns; the crawl date, domain, URL, and the content (full-text) of a given web page.') %>
+              <%= link_to('Web Page Text<br />'.html_safe + number_to_human_size(File.size(fulltext_path(@user.id, @collection.id, @collection.account))), user_collection_download_fulltext_path(@user.id, @collection.id, @collection.account), class: 'btn btn-primary', 'data-toggle': 'tooltip', 'data-placement': 'top', title: 'A csv file that contains six columns; the crawl date, domain, MIME type (web server, and Apache Tika), URL, and the content (full-text) of a given web page.') %>
```

Comment:
`six columns; the crawl date, domain, MIME type (web server, and Apache Tika), URL, and the content (full-text) of`

Might tweak the language a little bit, as I assume we're getting two MIME types above to make up the six columns. Maybe something like:

`six columns: the crawl date, domain, MIME type from the web server, MIME type as identified by Apache Tika, URL, and the content (full-text of each webpage)`